### PR TITLE
www.augmint.cc corrected

### PR DESCRIPTION
**Pull request by lbarjak**

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmintsim* ready!

Built with commit 6778edc5e0f23be010b1be54410e370b2eafca38

https://deploy-preview-49--augmintsim.netlify.com

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 # Augmint - Stable Digital Tokens - Simulator
 
-Augmint is a decentralised stable cryptocurrency on Ethereum. Read more about the concept on [wwww.augmint.cc](http://wwww.augmint.cc)
+Augmint is a decentralised stable cryptocurrency on Ethereum. Read more about the concept on [www.augmint.cc](http://www.augmint.cc)
```
Comment (szerintedmi) [inline README.md]: hoops, thank you.
If we are here then could we link with https:  please?

**Review by szerintedmi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lbarjak) [reply README.md]: Moment, please.

**Comment by szerintedmi** [comment]
@lbarjak : thanks for the https fix.
I can't merge as it has conflicts - latest actor UI additions are missing.  You need to pull from staging and merge with conflicts resolved.

**Review by szerintedmi — APPROVED** [review]

**Review by lbarjak — COMMENTED** [review]
ok.

**Pull request merged**